UnitySetup/UnitySetup.psm1

```diff
@@ -2823,8 +2823,19 @@ function Update-UnityPackageManagerConfig {
             Export-UPMConfig -UPMConfig $upmConfigs -tomlFilePaths $tomlFilePaths
         }
 
-        Write-Verbose "Summary"
-        Write-Output $upmConfigs
+        if ($upmConfigs) {
+            Write-Verbose "Summary"
+            $upmConfigs = $upmConfigs | ForEach-Object {
+                [PSCustomObject]@{
+                    ScopedURL = $_.ScopedURL
+                    Succeeded = -not [string]::IsNullOrEmpty($_.Auth)
+                }
+            } | Format-Table -AutoSize | Out-String
```

Comment:
Nit: a little strange to replace the table of objects with a string, thus changing the type.

Reply:
Yeah this isn't needed either - was part of a local iteration before I added the 'No changes were made' message.  Removed.